Library/Formula/zmqpp.rb

```diff
@@ -0,0 +1,26 @@
+class Zmqpp < Formula
+  desc "High-level C++ binding for zeromq"
+  homepage "https://zeromq.github.io/zmqpp/"
+  url "https://github.com/zeromq/zmqpp/archive/4.1.2.tar.gz"
+  sha256 "831ad02df64034268d910c30f9fb1b1e631ad810182951af9d7d622650831eb5"
+
+  depends_on "zeromq"
+
+  def install
+    ENV.cxx11
```

Comment:
Presumably also need a `needs :cxx11` one empty line underneath the `depends_on "zeromq"` line?